src/Stratis.Bitcoin.Features.PoA/PoANetwork.cs

```diff
@@ -14,6 +14,13 @@ public class PoANetwork : Network
         /// <summary> The default name used for the Stratis configuration file. </summary>
         private const string NetworkDefaultConfigFilename = "poa.conf";
 
+        /// <summary>Public keys of all federation members.</summary>
+        /// <remarks>
+        /// Blocks that are not signed with private keys that correspond
+        /// to public keys from this list are considered to be invalid.
+        /// </remarks>
+        public List<PubKey> FederationPublicKeys { get; }
```

Comment:
This should proabably go in PoaConsensusOptions no?

Reply:
Actually I am not so sure.

Reply:
No, that's not a part of consensus. 
We can have several PoA networks with same consensus rules but different federations. Sj it's a part of network settings, not consensus.